fabric-lifecycle-events-v1/src/main/java/net/fabricmc/fabric/mixin/event/lifecycle/MinecraftServerMixin.java

```diff
@@ -101,4 +101,14 @@ private void endResourceReload(Collection<String> collection, CallbackInfoReturn
 			return value;
 		}, (MinecraftServer) (Object) this);
 	}
+
+	@Inject(method = "save", at = @At("HEAD"))
+	private void startSave(boolean suppressLogs, boolean flush, boolean force, CallbackInfoReturnable<Boolean> cir) {
```

Comment:
Question: Are any of these params useful? Such as `flush` or `force`, im not 100% sure what they do without looking at the code.

Reply:
`force` is whether `save-off`ed servers should save. `flush`, see https://minecraft.wiki/w/Commands/save#save-all

Reply:
This kind of info is definitely worth passing, `force` at the very least.

Reply:
I was curious about the params, I thought they would be nice to have. wanted to discuss it. I can certainly pass this info